### ISSUE TITLE
Not able to drag an item

### ISSUE THREAD

**Issue by chiragbhagat** [swimlane/ngx-dnd#134]
I am using Angular 6.x and not able to drag an item on UI. Am I missing anything?

`<div class="ngx-dnd-container" ngxDroppable>
	<div class="ngx-dnd-item" ngxDraggable
		*ngFor="let item of documentList">
			{{item}}
	</div>
</div>`

**Comment by 0f0** [comment]
you shoud 
<div class="ngx-dnd-container" ngxDroppable="example-two">
            <div class="ngx-dnd-item" ngxDraggable *ngFor="let name of showStaff; let i=index">
              {{name?.name||name?.userId?.name}}
            </div>
          </div>

can you give me your code?

**Comment by 0f0** [comment]
<div class="ngx-dnd-container" ngxDroppable="example-two">
            <div class="ngx-dnd-item" ngxDraggable *ngFor="let name of showStaff; let i=index">
              {{name?.name||name?.userId?.name}}
              <i class="iconfont icon-msnui-close" style="margin-left: 33px" (click)="deletitem(i)"></i>
            </div>
          </div>
this is my code

**Comment by 0f0** [comment]
`<div class="ngx-dnd-container" ngxDroppable="example-two">
            <div class="ngx-dnd-item" ngxDraggable *ngFor="let name of showStaff; let i=index">
              {{name?.name||name?.userId?.name}}
              <i class="iconfont icon-msnui-close" style="margin-left: 33px" (click)="deletitem(i)"></i>
            </div>
          </div>`
this is my code

**Comment by 0f0** [comment]
`<div class="ngx-dnd-container" ngxDroppable="example-two">
            <div class="ngx-dnd-item" ngxDraggable *ngFor="let name of showStaff; let i=index">
              {{name?.name||name?.userId?.name}}
       
            </div>
          </div>`

this is my code

**Comment by gustavofscomp** [comment]
Any luck with this problem? Got the same situation!

<div class="columns" style="min-height: 300px; max-height: 300px;">
    <div class="left-column ngx-dnd-container" ngxDroppable="example" 
 (drop)="droppedToLeft($event)" (over)="drag(1)">
      <div class="ngx-dnd-item" ngxDraggable *ngFor="let item of leftColumnList">{{item}}</div>
    </div>

    <div class="right-column ngx-dnd-container" ngxDroppable="example" (drop)="droppedToRight($event)" (over)="drag(2)">
      <div class="ngx-dnd-item" ngxDraggable *ngFor="let item of rightColumnList">{{item}}</div>
    </div>
  </div>

**Comment by chiragbhagat** [comment]
Yes, I was able to resolve the issue for re-arranging order of my documents. I will share details with you soon. If urgent, you can email me directly at my id chiragbhagat in gmail and skype.

Thanks,
Chirag

**Comment by gustavofscomp** [comment]
Found the error, just some imports missing.
